### PR TITLE
Blob maker

### PR DESCRIPTION
**Pull request by CharlieWinters**
Hello! I figured I'd start with the blob maker app. 
One caveat, there seems to be a limitation when the app is served from localhost for the drag and drop functionality on the Miro side. I've added to error handling and documentation to inform the user of this. 
If you deploy to Glitch, you'll be able to use the full drag and drop functionality. To make things easier to deploy to glitch I've also hosted the code in it's own repo here: https://github.com/CharlieWinters/miro-blob-maker

### CODE REVIEW

File: examples/blob-maker/jsconfig.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "include": ["src", "typings"],
+  "exclude": ["node_modules"]
+}
```
Comment (ahmedelgabri) [inline examples/blob-maker/jsconfig.json:4]: Please do these changes.

```diff
{
+  "include": ["src"],
+  "compilerOptions": {
+    "typeRoots": ["./node_modules/@types", "./node_modules/@mirohq"]
+  },
  "exclude": ["node_modules"]
}
```

File: examples/blob-maker/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "blob-maker",
+  "version": "0.1.0",
+  "licence": "MIT",
+  "scripts": {
+    "dev": "npx supervisor server.js",
```
Comment (ahmedelgabri) [inline examples/blob-maker/package.json:6]: You can just call `supervisor` here directly since it's installed with the application.

**Review by ahmedelgabri — COMMENTED** [review]
@CharlieWinters the repo uses `yarn` not `npm`, so please run `yarn` to update the `yarn.lock` file & also remove `package-lock.json` file.

**Review by addisonschultz — APPROVED** [review]
Looks good to me @CharlieWinters! Thank you so much for adding the example. I've added in Ahmeds feedback, and cleaned up the following

- Added styles to app html
- Updated styles to match Mirotone
- Updated documentation
- Cleaned up logs and error handling

Overall nice work! I'll approve and merge.

**Pull request merged**